docs/context_standards/README.MD

```diff
@@ -305,6 +310,27 @@ The following is the format for a process.
     "Child": "LIST, of Child process objects"
 }
 ```
+
+## Host
+The following is the format for a host.
+```python
+"Host": {
+    "Domain": "STRING, Domain of the host.",
+    "Hostname": "STRING, Name of the host.",
+    "BIOVersion": "STRING, BIOS version of the host.",
+    "Blocked": "BOOLEAN, Whether the indicator is blocked, for example, in the firewall or blacklisted.",
+    "ID": "STRING, The unique ID within the tool retrieving the host.",
+    "DHCPServer": "STRING, DHCP Server.",
+    "IP": "STRING, IP Address of the host.",
+    "MACAddress": "STRING, Host MAC Address.",
+    "Memory": "STRING, Host memory.",
+    "Model": "STRING, Host model.",
+    "OS": "STRING, Host operating system.",
+    "OSVersion": "STRING, Host operating system version.",
+    "Processor": "STRING, Host processor.",
+    "Processors": "STRING, Host processors" 
```

Comment:
why do we have both single and plural of processor?
```suggestion
    "Processors": "STRING, Host processors." 
```

Reply:
Because the second field is the Processors number - I'll update the description to better reflect this